site/app/libraries/database/DatabaseQueries.php

```diff
@@ -4304,7 +4304,7 @@ public function getUsersNotificationSettings(array $user_ids) {
      * Gets All Parent Authors who this user responded to
      *
      * @param string $post_author_id current_user_id
-     * @param string $post_id        the parent post id
+     * @param string $post_id        the parent post idgetAllQueues
```

Comment:
why is getAllQueues appended to `id` here?

Reply:
whoops I think I was trying to do a ctrl f and accidentally typed in the file instead of the search box